modules/distr-tracing-removing-tempo-instance.adoc

```diff
@@ -0,0 +1,28 @@
+//Module included in the following assemblies:
+//
+//* distr_tracing_install/dist-tracing-tempo-removing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-removing-tempo-instance_{context}"]
+= Removing a {TempoName} instance using the web console
+
+[NOTE]
+====
+When deleting an instance that uses the in-memory storage, all data is permanently lost. Data stored in a persistent storage such as Elasticsearch is not be deleted when a {TempoName} instance is removed.
+====
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the name of the project where the Operators are installed from the *Project* menu, for example, `openshift-operators`.
+
+. Click the {TempoName} Operator.
+
+. Click the *Tempo* tab.
```

Comment:
This should be click on TempoStack tab. I'll leave screenshots of the TempoStack instance deletion and Tempo operator uninstallation page for reference.